protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1269,7 +1278,11 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 				if err != nil {
 					return err
 				}
-				parameters = append(parameters, queryParams...)
+				for _, queryParam := range queryParams {
+					if _, ok := notVisibleFields[queryParam.Name]; !ok {
```

Comment:
Does the update_mask only appear in query parameters? Does it never appear in the body unintentionally?

Reply:
I remember that the update_mask field does not appear in the body, I need to look at the code to confirm

Reply:
I didn't find that the body may appear to be filled with FieldMask